src/Trigger.hs

```diff
@@ -61,3 +64,10 @@ trigger session = do
       xs <- run
       success <- isSuccess <$> hspecPreviousSummary session
       return (success, xs)
+
+    whenHook hook hookName cont = (hook $ sessionHooks session) >>= \ case
+      ExitFailure _ -> do
+        let message = hookName <> " hook failed\n"
+        echo session message
```

Comment:
What do you think about handling error message in `callMbCommand`? This would allow for more meaningful feedback (say we could include the command in the error message).

Reply:
done